Make.inc

```diff
@@ -892,7 +892,7 @@ OPENBLAS_TARGET_ARCH:=ARMV7
 endif
 
 # on x86 make sure not to use 80 bit math when we want 64 bit math.
-ifeq ($(BUILD_ARCH),i386)
+ifeq (1,$(ISX86))
```

Comment:
Move it down to the other `ISX86` section and check for `BINARY`?

Reply:
what is `BINARY`?

Reply:
oh, the number of bits. Why check `BINARY`? We presumably want this on 64 bit...

Reply:
defined to `32` or `64` depending on the bitness.

Reply:
No, it's the default on x86_64, so no need to add the extra flag.